lms/djangoapps/grades/tests/test_models.py

```diff
@@ -140,6 +140,7 @@ def test_creation(self):
         self.assertEqual(expected_data, json.loads(vblocks.blocks_json))
         self.assertEqual(expected_json, vblocks.blocks_json)
         self.assertEqual(expected_hash, vblocks.hashed)
+        self.assertEqual(1, vblocks.version)
```

Comment:
Could you import the `VISUAL_BLOCKS_VERSION` variable here instead of using a magic number?